ln1_f4

```diff
@@ -0,0 +1,158 @@
+#!/usr/bin/python
+#coding: utf8
+#Програма повинна запускатися кожних 5хв (12 запусків за год) і записувати статистику використання ресурсів в БД, через годину 
+#дані з БД усереднюються та відправляються на електронку у вигляді таблиці з статистикою
+
+import smtplib
+import subprocess
+from email.mime.multipart import MIMEMultipart
+from email.mime.text import MIMEText
+
+data_string = [] #Усі дані зберігаються у форматі cpu_total,cpu_user,cpu_sys,cpu_idle,mem_total,mem_used,mem_free,mem_cached,hdd_total,hdd_used,hdd_free
+
+def get_raw_data(command): #функція виконує консольну команду та повертає результат
+    command_data = subprocess.Popen(command, stdout=subprocess.PIPE, shell=True)
+    (command_data, err) = command_data.communicate()
+    command_data = command_data.replace(",",".") #Заміна ком на крапки для коректного виконання float()
```

Comment:
https://www.python.org/dev/peps/pep-0008/#whitespace-in-expressions-and-statements